tox.ini

```diff
@@ -0,0 +1,6 @@
+[tox]
+envlist=py35
+
+[testenv]
+commands=py.test
+deps=pytest
```

Comment:
pyenv and tox itself is different tools

i use both. pyenv is to test a single python version (at once), and tox to test it all at once.